jol-benchmarks/src/main/java/org/openjdk/jol/GraphInstanceBench.java

```diff
@@ -46,7 +46,7 @@ public class GraphInstanceBench {
     public void setup() {
         arr = new Integer[size];
         for (int c = 0; c < size; c++) {
-            arr[c] = new Integer(c);
+            arr[c] = c;
```

Comment:
This boxing was intentional, please revert this hunk.

Reply:
I am revert this hunk,but I wonder why this is done on purpose.

Reply:
Because we want allocations to be cobbled together, not referenced from the integer cache.